src/app/components/shell/header/header.hbs

```diff
@@ -3,32 +3,200 @@
     <div class="skiptocontent">
         <a href="#maincontent">skip to main content</a>
     </div>
-
-    <nav class="nav">
-        <span class="flex-1-0 logo-wrapper">
+    <nav class="meta-nav" id="meta-nav">
+      <ul class="nav-menu">
+          <li class="nav-menu-item"><a href="/news">News</a></li>
+          <li class="nav-menu-item"><a href="/contact">Contact</a></li>
+          <li class="nav-menu-item"><a href="/about">About</a></li>
+          <li class="nav-menu-item"><a href="/give">Give</a></li>
+          <li class="nav-menu-item"><a href="/login">Login</a></li>
+      </ul>
+      <span class="logo-wrapper">
+          <a class="logo rice-logo" href="/" aria-label="Rice University">
+              <img src="/images/rice.png" alt="Rice University logo" width="74" height="32" />
+          </a>
+      </span>
+  </nav>
+  <nav class="nav" id="main-nav">
+        <span class="logo-wrapper">
             <a href="/" aria-label="Home Page">
                 <span class="logo os-logo">
-                    <img src="/images/logo.png" alt="OpenStax logo" height="40" />
-                </span>
-            </a>
-        </span>
+                  <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+                  	 viewBox="0 0 130.4 29.6" style="enable-background:new 0 0 130.4 29.6;" xml:space="preserve">
 
-        <span class="flex-1-0 hidden-desktop">
-            <button class="expand-nav">&#9776;</button>
-        </span>
+                  <symbol  id="Logo" viewBox="15 -60.7 200 45.7">
```

Comment:
Why put this in the HTML rather than as a separate SVG file?  It makes it kind of difficult to edit/read.